src/cmd/src/metasrv.rs

```diff
@@ -93,6 +93,8 @@ struct StartCommand {
     #[clap(long)]
     use_memory_store: bool,
     #[clap(long)]
+    disable_region_failover: bool,
```

Comment:
Prefer the name `enable_region_failover` and `true` by default.

Reply:
For the bool value `bool-option`, if you want to set it as true, the command for clap is `metasrv start --bool-option`. Conversely, the command is `metasrv start` if you want to set it as false.